.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
The commit history did not show the creation of the test.md file that is ignored as stated in the requirements